frontend/src/modules/entitieslist/components/Entity.js

```diff
@@ -130,12 +144,19 @@ export default class Entity extends React.Component<Props> {
                     className='status fa'
                     onClick={this.toggleForBatchEditing}
                 />
+                {parameters.search || this.areFiltersApplied() ? (
+                    <i
+                        className={'sibling-entities fas fa-arrows-alt-v'}
+                        title='Get more entities'
```

Comment:
Please replace text with the one used in the mockup: "Click to reveal surrounding strings".

And make it [localizable](https://github.com/mozilla/pontoon/tree/master/frontend#localization), i.e. wrap in the Localized element and create an entry for it in the .ftl file.